vue3/src/style.css

```diff
@@ -0,0 +1,26 @@
+:root {
+  font-family: 'Noto Sans', Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color: #213547;
+  background-color: #ffffff;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+#app {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+}
+
+a:hover {
```

Comment:
Looks like the example doesn't have any links or buttons - should we remove the CSS code for that here? If not, please use GCDS tokens for that as well.